### ISSUE TITLE
Remove Axes

### ISSUE THREAD

**Issue by TheDataLeek** [mpld3/mpld3#197]
I'm trying to create a network visualization, using NetworkX, Matplotlib, and mpld3. NetworkX creates a dictionary of node locations based on the matplotlib figure's axes instance, however once nodes have been placed the axes mean nothing. I've tried looking through documentation, examples, and previous issues for a solution to this, however I can't figure out how to completely remove the figure axes. The closest I've been able to get is a set of x/y axes without ticks and labels, in other words a straight line (as is seen in this example: http://mpld3.github.io/examples/random_walk.html).

To sum up, is there any way to completely remove the axes from the figure?
